lib/resque/failure/base.rb

```diff
@@ -47,7 +47,7 @@ def self.all(offset = 0, limit = 1)
       end
 
       # Iterate across failed objects
-      def self.each(offset = 0, limit = self.count)
+      def self.each(offset = 0, limit = self.count, queue = nil, class_name = nil)
```

Comment:
What exactly it does?.. I see an empty method.

Reply:
@ddiachkov 
I think it do nothing, while base.rb being inherited for other failure classes, but it was called with 4 arguments, so it raised exception wrong number of arguments.
I checked the next version (1.24.0) on Resque Repo and i found it changed too to : 
https://github.com/resque/resque/blob/v1.24.0/lib/resque/failure/base.rb#L50